openedx/features/caliper_tracking/tests/expected/edx.course.enrollment.activated.json

```diff
@@ -0,0 +1,41 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "id": "urn:uuid:2635b9dd-0061-4059-ac61-2718ab366f75",
+  "type": "Event",
+  "actor": {
+    "id": "edx.devstack.lms:18000/u/ali",
+    "type": "Person"
+  },
+  "action": "Activated",
+  "object": {
+    "id": "course-v1:edx+cs-101+2018",
+    "type": "CourseOffering",
+    "username": "ali"
```

Comment:
I believe `username` is not a valid field of `object` key according to Caliper. Verify this from the documentation. Also, it is unnecessary because the value of `username` is already present in `actor` key.

Reply:
yes it's true actually i misread your previous comment so that i moved it to the `object` now it has been fixed.